code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -87,6 +87,84 @@
 	hands_use_check = TRUE
 	sound = 'sound/mobs/humanoids/human/salute/salute.ogg'
 
+/datum/emote/living/carbon/human/slit
+	key = "slit"
+	key_third_person = "slits"
+	message = "drags a finger across their neck."
+	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/scratch_h
+	key = "scratch_h"
+	message = "scratches their head."
+	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/facepalm
+	key = "facepalm"
+	key_third_person = "facepalms"
+	message = "facepalms!"
+	hands_use_check = TRUE
```

Comment:
The rest are neat, but we already have facepalming via slapper - this shouldn't be a separate emote.